docs/tutorials/assets-tokens/withdrawing-assets.md

```diff
@@ -12,31 +12,36 @@ import NetworkParameter from '@site/src/components/NetworkParameter';
+Before beginning a withdrawal, it is worth confirming that there's no scheduled network restart or upgrade in the next 48 hours.
 
 * Withdrawals may be subject to a delay once started.
 * Any withdrawals that are left 'in progress' during a network restart or upgrade will not be visible in any UI or API request. **Your assets will be safe** but you will need to get the details of your transaction. See the instructions below for how to do this.
 :::
 
-The easiest way to withdraw assets is to use the Console trading interface, which will walk you through the process and notify you of progress. However, if you'd like to interact directly with Etherscan instead, use the instructions below.
+## Withdraw your trading assets
+If you want to withdraw assets from your general account, you won't need to take any extra steps.
 
-## What you need
-You'll need the following information available:
-* Vega public key you want to withdraw from
-* Vega Wallet connection details (wallet name, passphrase)
-* ERC-20 bridge logic address
-* Vega asset ID
-* Receiver Ethereum address for the withdrawal
+To remove all assets from your public key, you'll need to close all your open positions, cancel your liquidity commitments, if you have any, and redeem your vested staking and trading rewards.
 
-### Bridge address 
-The ERC-20 bridge address shown is for the Ethereum network that is compatible with the Vega network these docs are pointing to. 
+If you plan to stop using the key entirely, check if you have any [VEGA tokens](./withdrawing-vega-tokens.md#check-the-status-of-your-vega) connected to it.
 
-<EthAddresses frontMatter={frontMatter} show={["ERC20Bridge"]} />
+Console has a step-by-step process to move your assets out of Vega; this is the easiest way to withdraw them. If you want to use Etherscan instead, follow the [instructions](#withdraw-using-etherscan) below.
+
+## Withdraw your rewards
+Assets received as rewards may be locked for a time. Each epoch, a percentage of the locked amount begins vesting. Any amount that is vested is moved into a vested account. From there, you can redeem vested tokens by transferring them into your general account. Then, they can be withdrawn to an Ethereum address.
+
+1. You can check the status of all your assets using the [Explorer dApp](https://explorer.fairground.wtf) by providing your public key. 
+
+2. Any assets that are listed on Exlorer as in the "Vested account" need to be redeemed. Assets that are locked or vesting can't be moved *yet*. To redeem your vested rewards, go to the rewards page on [Console](https://vegafairground.eth.limo/#/rewards). From there you can transfer those tokens from your vested account into your general account. 
+
+  a. From the Governance dApp, you can click on Redeem on the wallet sidebar. This will take you to Console where you can transfer those tokens to your Vega general account.
```

Comment:
```suggestion
  a. From the Governance dApp, you can click on Redeem on the wallet sidebar. This will take you to Console where you can transfer those assets to your Vega general account.
```